clang/unittests/Parse/ParseHLSLRootSignatureTest.cpp

```diff
@@ -29,6 +29,9 @@ using namespace llvm::hlsl::rootsig;
 
 namespace {
 
+static const llvm::dxbc::RootSignatureVersion DefVersion =
+    llvm::dxbc::RootSignatureVersion::V1_1;
```

Comment:
I don't think this global makes the tests clearer. Maybe throw in a `using llvm::dxbc::RootSignatureVersion::V1_1` and just pass `V1_1` to the Parser constructors if you want to be concise?